src/plone/restapi/tests/test_serializer.py

```diff
@@ -395,10 +404,18 @@ def test_serialize_returns_site_root_common(self):
         self.assertIn("title", self.serialize(self.portal))
         self.assertIn("description", self.serialize(self.portal))
 
+    @unittest.skipIf(
+        PLONE_6,
+        "This test is only intended to run for Plone 6 and DX site root enabled",
+    )
     def test_serialize_returns_site_root_opt_in_blocks_not_present(self):
         self.assertEqual(self.serialize(self.portal)["blocks"], {})
         self.assertEqual(self.serialize(self.portal)["blocks_layout"], {})
 
+    @unittest.skipIf(
```

Comment:
@sneridagh just wondering. Don't we need a test that checks the Plone 6 root serialization? Or are the basics covered in another test already?